platform/admin/core/admin-core-impl/src/main/java/org/codice/ddf/admin/core/impl/AdminConsoleService.java

```diff
@@ -416,6 +426,18 @@ public void updateForLocation(
     }
   }
 
+  private Object sanitizeUIConfiguration(
+      String pid, String configEntryKey, Object configEntryValue) {
+    if (UI_CONFIG_PID.equals(pid)
+        && ("color".equals(configEntryKey) || "background".equals(configEntryKey))
+        && (Arrays.stream(ArrayUtils.toObject(String.valueOf(configEntryValue).toCharArray()))
+            .parallel()
+            .anyMatch(ILLEGAL_CHARACTER_SET::contains))) {
+      throw loggedException("Invalid UI Configuration");
```

Comment:
✏️  This message gets displayed to the admin who made the mistake. It should be more specific about what about the configuration was invalid. In other words maybe "Invalid UI Configuration: The color and background properties must only contain a color name".